pages/api/complete-visit.js

```diff
@@ -15,14 +15,14 @@ export default withContainer(
 
     checkIfAuthorised(userIsAuthenticatedResponse, res);
 
-    if (!body.callId) {
+    if (!body.callUuid) {
       res.status(400);
       res.end(JSON.stringify({ err: "callId must be present" }));
```

Comment:
error refers to callId. May want to change to callUuid.

Reply:
Done 🙂